src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -90,6 +91,9 @@ public Response buildErrorResponse(Throwable throwable) {
 
   public Response buildErrorResponse(HttpException exception) {
     final Response.ResponseBuilder responseBuilder;
+    Errors errors = exception.getErrors();
+    List<Error> errorList = errors.getErrors().stream().collect(toList());
```

Comment:
errors.getErrors().stream().collect(toList()) can be replaced just as: errors.getErrors()

Reply:
it should be fixed now.